### PR TITLE
feat: add createMarkdown script to generate rule's details docs base …

### LINKED ISSUE #149: generate skeleton docs for new rules
Being the base structure the following ⬇️⬇️

# Rule Name

Checks if ....

## ❓ Why it's good to follow this rule?

Content here...

## 😱 Examples of code for which this rule will throw a warning

Example of bad code...

Extra details of the example... (if applicable)

## 🤩 How to fix it?

Example of how to fix it 

Extra details of the example... (if applicable)

@rrd108 what do you think?

### PR DESCRIPTION
**Pull request by David-Pena**
### Summary
Create a script to generate the base template for a rules details page in the docs (markdown)

### Description
- Create `createMarkdown` script to generate the file in the docs folder
- Add script to `package.json` to trigger the script from above ⬆️
- Add `rulesDetailsSkeleton` file as base template
- Add a tip in the contribution guide in docs to showcase the new script

### Related Issues
Fixes #149 

### Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation update

### CODE REVIEW

File: src/generator/ruleDetailsSkeleton [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Checks if...
+
+## ❓ Why it's good to follow this rule?
+
+Content here...
+
+## 😱 Examples of code for which this rule will throw a warning
+
+Example of bad code...
```
Comment (rrd108) [inline src/generator/ruleDetailsSkeleton:11]: I would add the ::: warning tag here

File: src/generator/ruleDetailsSkeleton [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (rrd108) [inline src/generator/ruleDetailsSkeleton:12]: I would add a code block here with a todo comment

File: src/generator/ruleDetailsSkeleton [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Extra details of the example... (if applicable)
+
+## 🤩 How to fix it?
+
+Example of how to fix it...
```
Comment (rrd108) [inline src/generator/ruleDetailsSkeleton:17]: I would add the ::: tip tag here

and an empty script tag with a todo comment

**Review by rrd108 — CHANGES_REQUESTED** [review]

File: src/generator/ruleDetailsSkeleton (hunk 1 above)
Comment (David-Pena) [reply src/generator/ruleDetailsSkeleton]: Added in last commit

File: src/generator/ruleDetailsSkeleton (hunk 2 above)
Comment (David-Pena) [reply src/generator/ruleDetailsSkeleton:14]: Added in last commit

File: src/generator/ruleDetailsSkeleton (hunk 3 above)
Comment (David-Pena) [reply src/generator/ruleDetailsSkeleton]: Added in last commit

**Comment by David-Pena** [comment]
Ready to be reviewed again @rrd108

**Review by rrd108 — APPROVED** [review]

**Comment by rrd108** [comment]
Would not be a better file name: generateDoc

**Comment by David-Pena** [comment]
> Would not be a better file name: createDoc

That sounds like a good `good first issue` ... what do you think?

**Pull request merged**